### ISSUE TITLE
Pixie don't work with cgroup2

### ISSUE THREAD

**Issue by Emin3mU** [pixie-io/pixie#377]
**Describe the bug**
I deployed pixie with cgroup2, i got nothing when running script.
@oazizi000 found a critical error in the log below.
```
I1213 09:52:37.039113 32544 cgroup_path_resolver.cc:127] Auto-discovered CGroup base path: /sys/fs/cgroup/pids
E1213 09:52:37.214429 32544 cgroup_metadata_reader.cc:44] Failed to create path resolver. Falling back to legacy path resolver. [error = Not Found : Could not find self as a template.]
```
I change it to cgroup1, Pixie works.
It should not have the directory (/sys/fs/cgroups/pids) in cgroup2 environment. But I found that this directory exists, maybe its crio's bug.

**Logs**
[pixie_logs_20211213103207.zip](https://github.com/pixie-io/pixie/files/7759325/pixie_logs_20211213103207.zip)

**App information (please complete the following information):**
- Pixie version 
- K8s cluster version  1.22
- Node Kernel version  4.18
- Browser version

**Comment by noman-xg** [comment]
Hey @Emin3mU I'm having some trouble reproducing this error. Below are the details.

1- Cluster : minikube
2- Host OS: Ubuntu 20.04 
3- Cgroup version: v2 
4- Docker driver configured to use Cgroup driver ( also v2 ) 


Kelvin logs. 
```
Kelvin logs
 Defaulted container "app" out of: app, cc-wait (init), qb-wait (init)
 I20221027 17:20:05.286068     1 env.cc:47] Started: /app/src/vizier/services/agent/kelvin/kelvin
 Started external stacktrace collection signal processor thread
 I20221027 17:20:05.286262     1 kelvin_main.cc:109] Pixie Kelvin. Version: v0.12.7+Distribution.bf36058.202210202302.1.RELEASE.jenkins, id: 492bd8da-3d44-4f23-a1ae-143283575363
 I20221027 17:20:05.288744     1 manager.cc:156] Hostname: kelvin-9c89c8c99-wgh5k
 Server listening on 0.0.0.0:59300
 I20221027 17:20:44.790637     1 cgroup_path_resolver.cc:127] Auto-discovered CGroup base path: /sys/fs/cgroup/cpu,cpuacct
 I20221027 17:20:44.792351     1 cgroup_path_resolver.cc:130] Auto-discovered example path: /sys/fs/cgroup/cpu,cpuacct/kubepods.slice/kubepods-besteffort.slice/kubepods-besteffort-pod2c8e2c23_1c22_45f5_a22c_3942ffd841bf.slice/docker-0288b29d72b47f4931f12d6003724938ef02cba78bb2bc6040ac31a2a8d82be1.scope/cgroup.procs
 I20221027 17:20:44.792575     1 cgroup_path_resolver.cc:134] Auto-discovered template: /sys/fs/cgroup/cpu,cpuacct/kubepods.slice/kubepods-$2.slice/kubepods-$2-pod$0.slice/docker-$1.scope/cgroup.procs
 I20221027 17:20:44.792584     1 cgroup_metadata_reader.cc:41] Using path_resolver with configuration: template=/sys/fs/cgroup/cpu,cpuacct/kubepods.slice/kubepods-$2.slice/kubepods-$2-pod$0.slice/docker-$1.scope/cgroup.procs pod_id_separators=_ qos_spearator=-
 I20221027 17:20:44.817251     1 heartbeat.cc:140] Heartbeat ACK latency moving average: 2 ms
 I20221027 17:20:49.793473     1 metadata_state.cc:295] Didn't find pod UID 32549e66-d1fb-4965-87f6-7fe2c8423b7b for service px-operator/pixie-operator-index
 I20221027 17:20:49.793609     1 metadata_state.cc:295] Didn't find pod UID 8b3989cf-3969-46fe-88fc-d1f9bae14939 for service pl/pl-nats-mgmt
 I20221027 17:20:49.793965     1 metadata_state.cc:295] Didn't find pod UID b16b2046-3ce0-4c3f-9f79-7375c60a3caf for service kube-system/kube-dns
 I20221027 17:20:49.794461     1 metadata_state.h:111] Service CIDR updated to 10.96.0.0/12
 I20221027 17:20:52.025238     1 exec.cc:91] Queries in flight:

(Truncated)

 ```

I think I'm missing something here because even after configuring docker to use cgroup2, when i deploy pixie to minikube the pixie pods are still able to resolve the path and hence create the path template which should not be the case if I'm not mistaken.

Do you have any insights on this?

**Comment by noman-xg** [comment]
@zasgar Can this issue be closed now ?

**Comment by aimichelle** [comment]
Fixed by https://github.com/pixie-io/pixie/pull/635.

**Comment by benmathews** [comment]
What is the release schedule? When might this be in a release?

**Comment by oazizi000** [comment]
@benmathews This was addressed in 98189f6e939a9e7787ab31feca7ce3f7633a44ed, so you should already see cgroup v2 support. If you're still seeing issues, can you please start a new ticket so someone can investigate?
